src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <include resource="org/springframework/boot/logging/logback/base.xml" />
```

Comment:
You will need two loggers (as in application.yml):
```
<logger name="com.softserve.academy.spaced.repetition" level="info" />
<logger name="org.springframework" level="error" />
```

Reply:
For com.softserve.academy.spaced.repetition level="info" or level="error"?

Reply:
It up to you. I suggest - info.